pyensembl/reference_transcripts.py

```diff
@@ -115,6 +115,6 @@ def __contains__(self, transcript_id):
         # the pyfaidx __contains__ method requires an expensive list traversal
         # so cache the keys as a set for faster membership checks
         if self._fasta_keys is None:
-            keys = self.fasta_dictionary.keys()
+            keys = list(self.fasta_dictionary.keys())
```

Comment:
This annoys me as "wasteful" but I guess copying a list of transcript IDs won't take more than 10-30ms.